src/wp-includes/class-wp-meta-query.php

```diff
@@ -102,6 +102,7 @@ class WP_Meta_Query {
 	 * @since 5.1.0 Introduced `$compare_key` clause parameter, which enables LIKE key matches.
 	 * @since 5.3.0 Increased the number of operators available to `$compare_key`. Introduced `$type_key`,
 	 *              which enables the `$key` to be cast to a new data type for comparisons.
+	 * @since 6.0.0 Introduced `$compare_key_like_mode` and `$compare_like_mode` to `LIKE` queries
```

Comment:
Suggestion: add trailing period/full-stop for consistency.

Reply:
Done. Thank you. 
The other inline docs are aligned with the updated doc block format to my best knowledge. If I missed something, please let me know.